pygame/mixer_music.py

```diff
@@ -204,7 +204,12 @@ def queue(filename):
 def _endmusic_callback():
     global _current_music, _queue_music, _music_pos, _music_pos_time
     if _endmusic_event is not None and sdl.SDL_WasInit(sdl.SDL_INIT_AUDIO):
-        event.post(event.Event(_endmusic_event))
+        # Pygame doesn't do the same checks for this path as in
+        # event.post, and people rely on that, so we also duplicate
+        # the logic
+        event = ffi.new('SDL_Event*')
+        event.type = _endmusic_event
+        sdl.SDL_PushEvent(event)
```

Comment:
Would it be possible to add an option to `.post(...)` to not do the check? Maybe something like `.post(event, __skip_checks=True)`? I guess that makes sense if the odd behaviour is used elsewhere in pygame but not otherwise.